qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,340 @@
+                               initial_value='EXTERNAL',
+                               val_mapping={'CHANNEL_A': 0,
+                                            'CHANNEL_B': 1,
+                                            'EXTERNAL': 2,
+                                            'DISABLE': 3,
+                                            'CHANNEL_C': 4,
+                                            'CHANNEL_D': 5})
+            self.add_parameter(name='trigger_slope' + i,
+                               get_cmd=None,
+                               set_cmd=None,
+                               parameter_class=TraceParameter,
+                               label='Trigger Slope ' + i,
+                               unit=None,
+                               initial_value='TRIG_SLOPE_POSITIVE',
+                               val_mapping={'TRIG_SLOPE_POSITIVE': 1,
+                                            'TRIG_SLOPE_NEGATIVE': 2})
+            self.add_parameter(name='trigger_level' + i,
+                               get_cmd=None,
+                               set_cmd=None,
+                               parameter_class=TraceParameter,
+                               label='Trigger Level ' + i,
+                               unit=None,
+                               initial_value=140,
+                               vals=validators.Ints(0, 255))
+        self.add_parameter(name='external_trigger_coupling',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='External Trigger Coupling',
+                           unit=None,
+                           initial_value='DC',
+                           val_mapping={'AC': 1,'DC': 2})
+        self.add_parameter(name='external_trigger_range',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='External Trigger Range',
+                           unit=None,
+                           initial_value='ETR_5V',
+                           val_mapping={'ETR_5V': 0,'ETR_TTL': 2})
+        self.add_parameter(name='trigger_delay',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Trigger Delay',
+                           unit='Sample clock cycles',
+                           initial_value=0,
+                           vals=validators.Multiples(divisor=8, min_value=0))
+        self.add_parameter(name='timeout_ticks',
+                           get_cmd=None,
+                           set_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Timeout Ticks',
+                           unit='10 us',
+                           initial_value=0,
+                           vals=validators.Ints(min_value=0))
```

Comment:
Such a comment should be added:
# The card has two AUX I/O ports, which AUX 2 is only controlled by the driver. AUX 1 is controlled by the firmware. So,
# the user should use AUX 2 to control it via aux_io_mode and aux-io_param.